map_server/srv/LoadMap.srv

```diff
@@ -0,0 +1,3 @@
+string map_path
+---
+bool success
```

Comment:
Also, the right place for this message is probably nav_msgs, given that is where GetMap is, etc (although I do realize that is in common_msgs, meaning getting a review/release is likely going to take longer)